_data/documentation/0_17_1.yaml

```diff
@@ -13,15 +13,15 @@ docs:
     path: html/kroxylicious-proxy
   - title: Record Encryption quickstart
     description: Shows how to use the proxy to provide an encryption at rest solution
-      for Apache Kafka
+      for Apache Kafka.
```

Comment:
The source of these files is https://github.com/kroxylicious/kroxylicious/blob/0e87026aabf4027fe5172c7924b5a22f7a56a4bf/kroxylicious-docs/docs/record-encryption-quickstart/doc.yaml
so you need to make the fix there so it is applied to future versions.